src/main/resources/META-INF/chisel_at.cfg

```diff
@@ -0,0 +1 @@
+public-f net.minecraft.block.Block field_176227_L # blockState
```

Comment:
Yep, removed the reflection stuff for this field, as its easier to maintain the AT than using reflection for every block